### ISSUE TITLE
Unable to change GPT model

### ISSUE THREAD

**Issue by iamdangry** [justyns/silverbullet-ai#32]
Updating the SETTINGS yaml with an alternate GPT model other than the default of 3.5 doesn't appear to take effect.

Snippet from SETTINGS:
```yaml
ai:
  textModels:
  - name: gpt-4o
    provider: openai
    modelName: gpt-4o
```

Console output says it detects a change but the model doesn't change:
![image](https://github.com/justyns/silverbullet-ai/assets/46516977/40343d70-e4d0-4b12-bb7c-47a0dc29dced)

This is further confirmed by the API Usage Dashboard showing my API calls are to `GPT-3.5-turbo-0125` instead of `gpt-4o`
![image](https://github.com/justyns/silverbullet-ai/assets/46516977/2345353f-0613-40ae-b0a7-9b47cda0bce5)

I can update the image model fine via SETTINGS and the console logs the change correctly.

Of course, if I have done a dumb, please let me know.

**Comment by justyns** [comment]
Hey @iamdangry 

If you haven't already, you also need to run `AI: Select Text Model from Config` to actually select the model you want to use:

![image](https://github.com/justyns/silverbullet-ai/assets/215985/4d939930-ef0b-4fef-8557-a3f1c84b0df2)
![image](https://github.com/justyns/silverbullet-ai/assets/215985/a230cbab-2438-43a0-b66e-ecceaf7395dc)


I'm realizing now there probably isn't a good indication that you have to do that 😅 

The selected model is per client (similar to the dark mode setting in sb).  I think we can improve this though and have the default model change automatically if it's the only one in the list like in your screenshot

**Comment by iamdangry** [comment]
Ahhhh that did the trick. Thanks!

Odd that changing the image model in the settings updated automatically? Atleast that seemed to be what was happening based off the console output. Is that the expected behaviour? Think this is what confused me and I didn't bother to look in the command palette like a smart user 😂

**Comment by justyns** [comment]
Thanks @iamdangry, I'm glad that worked!  

I pushed a new update so if there's only a single text or image model in the settings, it'll always select it as the current model.

> Odd that changing the image model in the settings updated automatically? Atleast that seemed to be what was happening based off the console output. Is that the expected behaviour?

What you saw with the image model is the expected behavior, but I guess I messed something up with some backwards compatibility logic I had from before we had a list of `textModels` and it was just a single object.   I ended up removing that since it's not needed anymore
